diff -rupN giada-1.4.0-src.old/CMakeLists.txt giada-1.4.0-src/CMakeLists.txt
--- giada-1.4.0-src.old/CMakeLists.txt	2025-12-05 15:26:15.000000000 +0100
+++ giada-1.4.0-src/CMakeLists.txt	2026-03-24 15:20:40.077858846 +0100
@@ -564,7 +564,7 @@ list(APPEND INCLUDE_DIRS ${LIBRARY_RTMID
 include (FetchContent)
 FetchContent_Declare(
 	FLTK
-	SOURCE_DIR ${CMAKE_SOURCE_DIR}/src/deps/fltk)
+	SOURCE_DIR ${CMAKE_SOURCE_DIR}/src/deps/fltk EXCLUDE_FROM_ALL)
 
 set(FLTK_BUILD_GL ON CACHE BOOL "FLTK: enable OpenGL support")
 set(FLTK_BUILD_TEST OFF CACHE BOOL "FLTK: don't build tests")
@@ -766,7 +766,7 @@ endif()
 # Extra parameters for audio plug-ins support.
 # ------------------------------------------------------------------------------
 
-add_subdirectory(${CMAKE_SOURCE_DIR}/src/deps/juce)
+add_subdirectory(${CMAKE_SOURCE_DIR}/src/deps/juce EXCLUDE_FROM_ALL)
 
 list(APPEND LIBRARIES 
 	juce::juce_audio_utils
