diff -rupN giada-1.2.1.old/CMakeLists.txt giada-1.2.1/CMakeLists.txt
--- giada-1.2.1.old/CMakeLists.txt	2025-07-25 15:48:15.684170500 +0200
+++ giada-1.2.1/CMakeLists.txt	2025-07-25 17:25:58.958027311 +0200
@@ -546,7 +546,7 @@ list(APPEND INCLUDE_DIRS ${LIBRARY_RTMID
 include (FetchContent)
 FetchContent_Declare(
 	FLTK
-	SOURCE_DIR ${CMAKE_SOURCE_DIR}/src/deps/fltk)
+	SOURCE_DIR ${CMAKE_SOURCE_DIR}/src/deps/fltk EXCLUDE_FROM_ALL)
 
 set(FLTK_BUILD_GL ON CACHE BOOL "FLTK: enable OpenGL support")
 set(FLTK_BUILD_TEST OFF CACHE BOOL "FLTK: don't build tests")
@@ -745,7 +745,7 @@ endif()
 # Extra parameters for audio plug-ins support.
 # ------------------------------------------------------------------------------
 
-add_subdirectory(${CMAKE_SOURCE_DIR}/src/deps/juce)
+add_subdirectory(${CMAKE_SOURCE_DIR}/src/deps/juce EXCLUDE_FROM_ALL)
 
 list(APPEND LIBRARIES 
 	juce::juce_audio_utils
